scheduler/src/cook/ratelimit.clj

```diff
@@ -0,0 +1,36 @@
+;;
+;; Copyright (c) Two Sigma Open Source, LLC
+;;
+;; Licensed under the Apache License, Version 2.0 (the "License");
+;; you may not use this file except in compliance with the License.
+;; You may obtain a copy of the License at
+;;
+;;  http://www.apache.org/licenses/LICENSE-2.0
+;;
+;; Unless required by applicable law or agreed to in writing, software
+;; distributed under the License is distributed on an "AS IS" BASIS,
+;; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+;; See the License for the specific language governing permissions and
+;; limitations under the License.
+;;
+(ns cook.ratelimit
+  (:require [clojure.tools.logging :as log]
+            [cook.config :refer [config]]
+            [mount.core :as mount]
+            [ratelimit]))
+
+(defn create-job-submission-rate-limiter
+  [config]
+  (let [{:keys [settings]} config
+        {:keys [rate-limit]} settings
+        {:keys [expire-minutes job-submission]} rate-limit]
+    (log/error "Trying mount " rate-limit)
```

Comment:
Why is this an error?

Reply:
Its not. Diagnostic logging left over from getting mount to work.